client/app/pages/dashboards/public-dashboard-page.js

```diff
@@ -44,6 +44,8 @@ const PublicDashboardPage = {
             if (!isSafe) {
               error.errorMessage = 'This query contains potentially unsafe parameters and cannot be executed on a publicly shared dashboard.';
             }
+
+            throw error;
```

Comment:
Will this make the whole dashboard load blow up?

Reply:
Nope, and this is not really a part of this PR :)

Anyway, this is removed in #3990